lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -7,13 +7,13 @@ defmodule Kernel.SpecialForms do
   `alias/2`, `case/2`, etc). The macros `{}` and `<<>>` are also special
   forms used to define tuple and binary data structures respectively.
 
-  This module also documents Elixir's pseudo variables (`__ENV__`,
-  `__MODULE__`, `__DIR__` and `__CALLER__`). Pseudo variables return
+  This module also documents Elixir's pseudo variables (`__ENV__/0`,
+  `__MODULE__/0`, `__DIR__/0` and `__CALLER__/0`). Pseudo variables return
```

Comment:
@josevalim so we skip the part about not being able to assign values. (since we no longer call them variables)